types/altv-server.d.ts

```diff
@@ -199,22 +227,37 @@ declare module "alt-server" {
   }
 
   export class Player extends Entity {
+    /*
+     * Returns all connected players to the server.
+     */
     public static all: Array<Player>;
     public armour: number;
+    /*
+     * The current selected weapon of the player.
```

Comment:
With a comment formulated like that it could also mean the weapon selected in weapon wheel